bambi/models.py

```diff
@@ -387,20 +400,38 @@ def _add_response(self, response, prior=None, family="gaussian", link=None):
             defined in the current backend; typically this will include at least ``'identity'``,
             ``'logit'``, ``'inverse'``, and ``'log'``), or a callable that takes a 1D ndarray or
             theano tensor as the sole argument and returns one with the same shape.
+        priors : dict
+            Optional dictionary with specification of priors for the parameters in the family of
+            the response. Keys are nuisance parameters in the family (i.e. they cannot be equal to
```

Comment:
nitpick: Not sure about the term "nuisance parameter", could we use "dispersion parameter"? or some other alternative.

Reply:
Yeah, I get the intended use. When dispersion does not apply? maybe we can call it the "~mean" parameter :-P

Reply:
Maybe in the negative binomial family? It uses mu and alpha, and alpha plays both in the shape and scale parameter, see [here](https://github.com/pymc-devs/pymc3/blob/d7172c0a1a76301031d1b3b411d00643c416a0c4/pymc3/distributions/discrete.py#L863-L873). But I understand your point. Saying "dispersion" is more intuitive and familiar for most people, but it is not accurate for all GLMs we may implement

Reply:
Yes you are right, but in that case you can think of alpha as the overdispersion parameter, right?

Reply:
Maybe "other parameters than the mean", is not that bad

Reply:
I agree on the last suggestion